src/NBitcoin/BIP9DeploymentsArray.cs

```diff
@@ -1,20 +1,51 @@
-﻿using System;
-
-namespace NBitcoin
+﻿namespace NBitcoin
 {
-    public class BIP9DeploymentsArray
+    /// <summary>
+    /// Interface for recording deployment parameters and returning deployment flags.
+    /// </summary>
+    public interface IBIP9DeploymentsArray
+    {
+        void GetFlags(int deployment, out ScriptVerify scriptFlags, out Transaction.LockTimeFlags lockTimeFlags);
```

Comment:
Spaces between lines, also can you document those properties as this is an interface

Reply:
Done.